src/__tests__/relative.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { relative } from '../';
+
+describe('relative', () => {
+  it('handles URLs', () => {
+    expect(relative('http://stoplight.io/bar/foo', 'http://stoplight.io/')).toEqual('bar/foo');
+    expect(relative('http://stoplight.io/bar/foo/baz', 'http://stoplight.io/bar/z/x')).toEqual('../foo/baz');
```

Comment:
Add some examples that should throw errors. E.g. two URLs with different origins.

Reply:
It does not throw when origins are different :man_shrugging:

Reply:
Frankly I think that's ok.

Reply:
Computing a relative URL from two different origins ought to throw an error, don't you think?